mapdata/WarcraftLegacies/UnitData/Skin/1362374766.json

```diff
@@ -23,7 +23,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Summoning site for minor demons.|r|n|cff99b4d1Can only build 2 before the Legion has been summoned. 12 post-summon.|n|cfff5962dTrains:|R Fel Guard, Burning Archer, Infernal|n|cfff5962dResearches:|R War Plating"
+      "Value": "Summoning site for minor demons.|r|n|cff99b4d1Can only build 3 before the Legion has been summoned. 12 post-summon.|n|cfff5962dTrains:|R Fel Guard, Burning Archer, Infernal|n|cfff5962dResearches:|R War Plating"
```

Comment:
I believe this tooltip is wrong; the limit after summoning the Legion would be 13, since that quest adds 10 to the total.

Reply:
Oh true  forgot to adjust the post summoning number

Reply:
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/47048283/dbb6289e-df4f-4ccd-95ad-fafb72846c46)
 the quest set it to 12, so it shold be fine!